babel.config.js

```diff
@@ -8,6 +16,9 @@ module.exports = function (api) {
       {
         autoLabel: 'dev-only',
         labelFormat: '[local]',
+        'preset-env': {
+          targets: browsersList,
```

Comment:
Would also be nice to have [`useBuiltins: 'usage'`](https://babeljs.io/docs/en/babel-preset-env#usebuiltins) here I think

Reply:
Ah, yes. Will do.

Reply:
And this was the reason why I needed the polyfil file. Thanks. Removed it and now will use babel 🥳